packages/buckets/package.json

```diff
@@ -2,8 +2,9 @@
   "name": "@textile/buckets",
   "version": "4.1.0",
   "description": "Web-gRPC wrapper client for managing Textile Buckets.",
-  "main": "dist/index",
-  "types": "dist/index",
+  "main": "dist/cjs/index.js",
```

Comment:
all package.json files have been updated with this. 1) more explicit path to specific files, 2) types only referenced in cjs path, though they are also build (as a single file) in the esm folder. module refers to dist/esm/* so that builders/packagers know where to look. in node, if someone is using es modules, it'll link to esm, otherwise, main. main is always the fallback, so older build systems should remain unaffected